release-team/role-handbooks/documentation-guides/update-release-docs-new.md

```diff
@@ -1,88 +1,180 @@
+
+    - Which features need user-facing docs
+    - Feature owners (and their GitHub IDs)
+    - The SIG associated with each feature (useful for obtaining technical reviews)
+    - Links to docs PRs opened for each feature
+
+### Docs starting tasks
+
+1. Create a pull request for the release.
+
+- Open a pull request against `master` with a new branch that follows the naming convention `release-X.Y`.
+
+    For example, the [Release 1.11](https://github.com/kubernetes/website/pull/8046) PR uses the branch `release-1.11`.
+
+    This release pull request (also known as a _megabranch_) serves as the base for individual, component feature PRs of the release. A megabranch lets you bundle and merge multiple PRs simultaneously.
+
+    Use previous release PRs as a template. For example: https://github.com/kubernetes/website/pull/8046
+
+- Add the label `do-not-merge/hold` to the PR.
+
+2. Add a tracking comment to the megabranch PR.
+
+Use this placeholder text:
+
+```
+## Release X.Y megabranch
+
+PR | Based on `release-X.Y`? | Tech LGTM | Docs approved | Merged
+---|---|---|---|---
+| | | |
+```
+
+    A tracking comment helps track the identity and status of component PRs in the megabranch.
+
+    For example: https://github.com/kubernetes/website/pull/8046#issuecomment-380226165
+
+2. Update the Netlify _vnext staging_ site to use the new branch:
+
+- Login to [Netlify](https://app.netlify.com/) and navigate to the Sites tab.
+
+- Open the settings for the [kubernetes-io-vnext-staging](https://app.netlify.com/sites/kubernetes-io-vnext-staging/settings) site.
+
+- Edit the Deploy settings and change "Branch" to the new version branch, for example: `release-1.11`.
+
+- Save the change and verify that the change is live at [https://kubernetes-io-vnext-staging.netlify.com/](https://kubernetes-io-vnext-staging.netlify.com/).
+
+- Update the [pull request template](https://github.com/kubernetes/kubernetes.github.io/blob/master/.github/PULL_REQUEST_TEMPLATE.md) with the version in development. For example: https://github.com/kubernetes/website/pull/8057
 
-*   Generate reference docs and place them in kubernetes/website. See [PR 6366](https://github.com/kubernetes/website/pull/6366).
-*   Update reference page for feature gates. See [PR 6364](https://github.com/kubernetes/website/pull/6364).
+## Continue the release cycle
+
+### Track PRs
+
+- Follow the feature spreadsheet.
+
+    Keep the feature tracking spreadsheet up to date with review progress and merge status for each documentation PR. For example: [Kubernetes Features OSS tracking board (1.11 release)](https://docs.google.com/spreadsheets/d/16N9KSlxWwxUA2gV6jvuW9N8tPRHzNhu1-RYY4Y0RZLs/edit#gid=0)
+
+- Update documentation PR progress in the megabranch PR's tracking comment. For example, [Release 1.11 (tracking comment)](https://github.com/kubernetes/website/pull/8046#issuecomment-380226165)
```

Comment:
Sorry if this isnt clear just to me. Looking ay the specific comment linked here I dont see any status update except for the placeholder text. Is there any other PR or comment we can point to here that explains this point better?